ci-operator/step-registry/openshift/e2e/libvirt/analyze-e2e-failure/openshift-e2e-libvirt-analyze-e2e-failure-commands.sh

```
@@ -215,19 +215,51 @@ TOKENS_JSON=$(grep '"type":"result"' "${CLAUDE_JSON}" 2>/dev/null \
+    -e 's/(aws_secret_access_key[":= ]+)[A-Za-z0-9\/+=]{40}/\1[REDACTED-AWS-SECRET]/g' \
+    -e 's/(Bearer )[A-Za-z0-9._~+\/=-]+/\1[REDACTED-TOKEN]/g' \
+    -e 's/gh[pousr]_[A-Za-z0-9_]+/[REDACTED-GITHUB-TOKEN]/g' \
+    -e 's/eyJ[A-Za-z0-9_-]*\.eyJ[A-Za-z0-9_-]*\.[A-Za-z0-9_-]*/[REDACTED-JWT-TOKEN]/g' \
+    -e "s/(password[\":= ]+)[^ \"']+/\\1[REDACTED-PASSWORD]/gi" \
+    -e "s/(api[_-]?key[\":= ]+)[^ \"']+/\\1[REDACTED-API-KEY]/gi" \
+    -e "s/(token[\":= ]+)[A-Za-z0-9._~+\/-]+=*/\\1[REDACTED-TOKEN]/gi"
+}
+
+REPORT="${ARTIFACT_DIR}/failure-analysis.html"
+if [[ -f "${REPORT}" ]]; then
+  REDACTED="${REPORT}.redacted"
```

Shellcheck (0.11.0)

[error] This is a unicode space. Delete and retype it.

(SC1018)